### PR TITLE
Sscs 8518 uc integration tests

### PR DESCRIPTION
**Pull request by michaellavelle**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-8518

### Change description ###

Adding UC integration tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java [hunk 1]
```diff
@@ -261,4 +265,1548 @@ public void callToMidEventPreviewFinalDecisionCallback_willPreviewTheDocumentFor
...
+
+        assertEquals(1, result.getErrors().size());
+        assertEquals("You have specified that the appeal is refused and specified that Support Group Only Appeal does not apply, but have answered Yes for the Schedule 8 Paragraph 4 question and submitted an unexpected answer for the Schedule 7 Activities question. Please review your previous selection.", result.getErrors().iterator().next());
+
+        assertEquals("http://dm-store:5005/documents/7539160a-b124-4539-b7c1-f3dcfbcea94c", result.getData().getWriteFinalDecisionPreviewDocument().getDocumentUrl());
+        verifyNoInteractions(generateFile);
+    }
+
+    @Test
+    public void callToMidEventPreviewFinalDecisionCallback_willPreviewTheDocumentForEsaNonWcaScenario10Refused() throws Exception {
```
Comment (chrisdavecm) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java:1485]: Uc

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java [hunk 2]
```diff
@@ -224,17 +228,17 @@ public void callToMidEventPreviewFinalDecisionCallback_willPreviewTheDocumentFor
...
-        assertEquals(1, payload.getEsaSchedule2Descriptors().size());
-        assertEquals(15, payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerPoints());
-        assertEquals("a", payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerLetter());
-        assertEquals("Cannot, unaided by another person, either: (i) mobilise more than 50 metres on level ground without stopping in order to avoid significant discomfort or exhaustion; or (ii) repeatedly mobilise 50 metres within a reasonable timescale because of significant discomfort or exhaustion.", payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerValue());
-        assertEquals("1. Mobilising unaided by another person with or without a walking stick, manual wheelchair or other aid if such aid is normally or could reasonably be worn or used.", payload.getEsaSchedule2Descriptors().get(0).getActivityQuestionValue());
-        assertEquals("1", payload.getEsaSchedule2Descriptors().get(0).getActivityQuestionNumber());
-        Assert.assertNotNull(payload.getEsaNumberOfPoints());
-        assertEquals(15, payload.getEsaNumberOfPoints().intValue());
+        assertEquals(true, payload.isUcIsEntited());
```
Comment (chrisdavecm) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java:231]: should this be entitled?

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java (hunk 1 above)
Comment (michaellavelle) [reply src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java]: Have now fixed

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java (hunk 2 above)
Comment (michaellavelle) [reply src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java]: No, it shouldn't - good spot Chris - thanks!   Have now changed this test

**Review by chrisdavecm — APPROVED** [review]

**Pull request merged**